Formula/proguard.rb

```diff
@@ -1,8 +1,8 @@
 class Proguard < Formula
   desc "Java class file shrinker, optimizer, and obfuscator"
   homepage "https://www.guardsquare.com/en/products/proguard"
-  url "https://github.com/Guardsquare/proguard/releases/download/v7.2.0/proguard-7.2.0.tar.gz"
-  sha256 "829a3420e52f49d5d2b751b5d9ab6f543ff503a87ffd835014222b40250c0a6d"
+  url "https://github.com/Guardsquare/proguard/releases/download/v7.2.1/proguard-7.2.1.tar.gz"
+  sha256 "c3a7062d42d2388445c4b699a131cae3f1c3c7b91834db0fbd4d964bc6376e4b"
   license "GPL-2.0"
```

Comment:
Upstream code headers include 
```
 * This program is free software; you can redistribute it and/or modify it
 * under the terms of the GNU General Public License as published by the Free
 * Software Foundation; either version 2 of the License, or (at your option)
 * any later version.
```

```suggestion
  license "GPL-2.0-or-later"
```